packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -33,12 +39,24 @@ export interface ExpandableSectionProps extends React.HTMLProps<HTMLDivElement>
   isWidthLimited?: boolean;
   /** Flag to indicate if the content is indented */
   isIndented?: boolean;
+  /** Determines the variant of the expandable section. */
+  variant?: 'default' | 'truncate';
+  /** @beta Truncates the expandable content to the specified number of lines. */
+  truncateMaxLines?: number;
```

Comment:
We should have default for this prop. Right now if you leave it out there is an error.
@mcarrano What should the default value be?

Reply:
That's hard to say, but lets go with 3.

Reply:
FWIW, the [default from core is already `3`](https://github.com/patternfly/patternfly/blob/50c8990653214df3ea2edf0b343fd34d053b1b30/src/patternfly/components/ExpandableSection/expandable-section.scss#L51), and it comes with the `.pf-m-truncate` class - you don't have to define the custom property. I'm guessing for consistency, it's probably best to use the default from core - and if we want to change that default, we would change it in core? So maybe this prop could continue to be optional for the user and we only set the custom property if `truncateMaxLines` is passed? https://github.com/patternfly/patternfly-react/blob/bb08fa4423d16458fe2104bf67f76807cc98100d/packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx#L57

Reply:
Currently my local build is using a default of 3 for `truncateMaxLines` and it looks to work well. So basically you would pass in `variant="truncate"` and it'll default to clamping to 3 lines. From there you can customize the truncate amount with `truncateMaxLines={<number other than 3>}`

I agree that with this approach it does make it clearer that truncate is a separate variant like Titani mentioned in a previous comment, vs only having `truncateMaxLines` it's not really clear it's a completely separate variant. I'm fine with either having both a `variant` and `truncateMaxLines` prop, or just using a `truncateMaxLines` prop, though if there's no strong feelings one way or the other I can push what I currently have and we can tweak it as-needed down the line.

Reply:
I think we should rename `truncateMaxLines` to `customTruncateMaxLines`. I still do not know if the prop should exist on its own.  If you do separate it out, i would specify that it is only applicable. if variant is "truncate".